crates/trie/sparse/benches/root.rs

```diff
@@ -201,6 +203,7 @@ fn calculate_root_from_leaves_repeated(c: &mut Criterion) {
                                         .update_leaf(
                                             Nibbles::unpack(key),
                                             alloy_rlp::encode_fixed_size(value).to_vec(),
+                                            &DefaultBlindedProvider,
```

Comment:
I find this syntax a bit odd, but since this is only just for testing, not worth adding a default + non_exhaustive imo, so this is fine